python/app/dialog.py

```diff
@@ -212,10 +214,15 @@ def create_ticket(self):
             ticket_body += "\n### Environment Variable\n{}".format(env_info)
 
         try:
+            if self._app.get_setting("project_id", ""):
```

Comment:
```python
project_id = self._app.get_setting("project_id", "")
project = {'type': 'Project', 'id': project_id}
if not project_id:
    project = self._app.context.project
```

Reply:
Honestly there is nothing wrong with what you wrote, I just think reads better this way.